maven-plugin/src/main/java/com/spotify/missinglink/maven/CheckMojo.java

```diff
@@ -128,6 +129,14 @@
   protected List<String> includeCategories = new ArrayList<>();
 
   /**
+   * Include dependencies with the following scopes in conflict checks. Default is "compile, test".
+   */
+  // NOTE: couldn't find a way to explicitly specify a default set of strings to include, so
+  // managing that explicitly in the execute() method below.
+  @Parameter(property = "missinglink.includeScopes", defaultValue = "compile, test")
```

Comment:
the `defaultValue` value seems to conflict with the comment above - is the comment outdated or does the `defaultValue` part not work?